internal/app/machined/pkg/runtime/v1alpha1/bootloader/bootloader.go

```diff
@@ -27,6 +28,9 @@ type Bootloader interface {
 	Revert(disk string) error
 	// RequiredPartitions returns the required partitions for the bootloader.
 	RequiredPartitions() []partition.Options
+
+	// KexecLoad does a kexec using the bootloader.
```

Comment:
```suggestion
	// KexecLoad does a kexec using the config from bootloader.
```

Maybe another rephrase, but the point is bootloader isn't used, but only its configs

Reply:
I would say we need to phrase that `KexecLoad` emulates boot as if it was done by the bootloader - we pick the entry to boot based on bootloader information, and then do the boot as if the machine was rebooted